include/roboteam_ai/stp/new_tactics/TestTactic.h

```diff
@@ -9,9 +9,9 @@
 namespace rtt::ai::stp {
 
 class TestTactic : public Tactic {
-        Status onInitialize() noexcept override;
-        Status onUpdate(TacticInfo const& info) noexcept override;
-        Status onTerminate() noexcept override;
+        void onInitialize() noexcept override;
+        Status onUpdate(SkillInfo const& info) noexcept override;
```

Comment:
Should be Tactic Info

```suggestion
        Status onUpdate(TacticInfo const& info) noexcept override;
```